### ISSUE TITLE
Update data_formats.rst when File Converter branch merged (Trac #709)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#833]
When Lewis' File Converter branch is merged and working data_formats.rst will need updating for data in separate I and Q files.

Migrated from http://trac.sasview.org/ticket/709
```json
{
    "status": "closed",
    "changetime": "2016-10-07T19:49:13",
    "_ts": "2016-10-07 19:49:13.112657+00:00",
    "description": "When Lewis' File Converter branch is merged and working data_formats.rst will need updating for data in separate I and Q files.",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2016-10-06T02:48:45",
    "component": "SasView",
    "summary": "Update data_formats.rst when File Converter branch merged",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "smk78",
    "type": "task"
}
```

**Comment by smk78** [comment]
Trac update at `2016/10/07 00:50:45`: 
* **smk78** changed owner from "" to "smk78"
* **smk78** changed status from "new" to "assigned"

**Comment by smk78** [comment]
Trac update at `2016/10/07 19:49:13`: 
* **smk78** commented:

In changeset 48b8f6d68e3c55ef89958eb0b7b56b22ea3d63d0:

```text
#!CommitTicketReference repository="sasview" revision="48b8f6d68e3c55ef89958eb0b7b56b22ea3d63d0"
Updated with SESANS format by Wim, then thoroughly edited by Steve.
Information on NxcanSAS and File Converter added too. Closes #833
```


* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "assigned" to "closed"
